### PR TITLE
Add schemas for non-lossy BQ sink

### PR DESCRIPTION
**Pull request by relud**
Checklist for reviewer:

- [x] Scan the PR and verify that no changes (particularly to `.circleci/config.yml`) will cause environment variables (particularly credentials) to be exposed in test logs

### CODE REVIEW

File: schemas/metadata/error/error.1.schema.json [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    "geo_city": {
+      "type": "string"
+    },
+    "geo_country": {
+      "type": "string"
+    },
+    "geo_subdivision1": {
+      "type": "string"
+    },
+    "geo_subdivision2": {
```
Comment (relud) [inline schemas/metadata/error/error.1.schema.json]: @whd the deployed schemas are currently missing the `geo_` and `user_data_` prefixed fields

File: schemas/metadata/error/error.1.schema.json [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+      "type": "string"
+    },
+    "host": {
+      "type": "string"
+    },
+    "method": {
+      "type": "string"
+    },
+    "payload": {
+      "format": "bytes",
```
Comment (relud) [inline schemas/metadata/error/error.1.schema.json]: @acmiyaguchi i'm unclear on whether the transpiler already does the right thing, or we need to add supports for the bytes format somehow

File: schemas/metadata/error/error.1.schema.json (hunk 1 above)
Comment (whd) [reply schemas/metadata/error/error.1.schema.json]: ACK

**Comment by whd** [comment]
One thing to note here is that if we're planning on using the standard transpiler to convert these into BQ schemas (and then wind up in the generated-schemas branch), we're going to also need to update MSG to that effect. Currently we [simply remove them](https://github.com/mozilla/mozilla-schema-generator/blob/master/bin/schema_generator.sh#L92).

The BQ table management (WIP [here](https://github.com/mozilla-services/cloudops-infra/tree/data_bq_storage_api/projects/data-shared/tf/modules/bigquery)) for these tables will have special case logic since they are generated by "pipeline family", which isn't a concept propagated to this repository.

**Comment by relud** [comment]
filed #373 to handle that

**Pull request merged**